cli/test/smokehouse/test-definitions/fps-max.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const MAX_TEXTURE_SIZE = process.platform === 'linux' ? 8191 : 16383;
```

Comment:
You got 8191 on some linux machine you have on-hand?

I imagine this test will be incredibly GPU specific. And is it really just testing `pageFunctions.getMaxTextureSize`?

Reply:
> You got 8191 on some linux machine you have on-hand?

Yes, my cloudtop.

> I imagine this test will be incredibly GPU specific. And is it really just testing pageFunctions.getMaxTextureSize?

Yes. We could also change the expectation to just accept both.

Reply:
I'm pretty sure this is gonna give different values on google3 (it'd be whatever swiftshader provides as the max texture size, and who knows if that's the same as if what swiftshader on our local machines would be). But we can skip this smoke test if needed in LR.

But I am hesitant to keep this smoke test because it seems guaranteed to fail on anything other than our exact machines.

Reply:
Also, since it's a really roundabout way to test that page function, I'd much prefer a more direct test of that function.

Reply:
Sorry I didn't mean that this entire test was just testing the page function. I only meant that `MAX_TEXTURE_SIZE` is supposed to represent the result of that function on different machines.

This smoke also ensures that the `Page.captureScreenshot` CDP function actually returns data if the page is super tall.